sklearn/svm/base.py

```diff
@@ -907,7 +907,7 @@ def _fit_liblinear(X, y, C, fit_intercept, intercept_scaling, class_weight,
     # on 32-bit platforms, we can't get to the UINT_MAX limit that
     # srand supports
     n_iter_ = max(n_iter_)
-    if n_iter_ >= max_iter and verbose > 0:
+    if n_iter_ >= max_iter:
```

Comment:
It would be great if you could estimate how much code is affected by this change. The best thing to do would be to add tests for the relevant estimators that they get a ConvergenceWarning.